ClosedXML/Excel/PivotTables/PivotStyleFormats/XLPivotStyleFormatEnums.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace ClosedXML.Excel
+{
+    [Flags]
+    public enum XLPivotStyleFormatElement
+    {
+        None = 0,
+        Label = 1,
+        Data = 2,
```

Comment:
```
None  = 1 << 0,
Label = 1 << 1,
Data  = 1 << 2,
```
This way chances someone will accidentally add the next flag with value 3 instead of 4 are lower.

Reply:
Yes, except `None = 0` should remain. `1 << 0 == 1`

Reply:
Ah, of course